cmd/version.go

```diff
@@ -20,11 +26,60 @@ var versionCmd = &cobra.Command{
 To see if there is a more recent version available, call the command with the
 --latest flag.
 	`,
+
 	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Println(Version)
+		o := currentVersion()
```

Comment:
Do we need to create this variable `o` here, could this be `fmt.Println(currentVersion())`?
Assigning `currentVersion()` to `o` makes me think it's used later on.